tests/e2etest/GenericTest/src/main/java/org/astarteplatform/devicesdk/tests/e2etest/GenericAggregateDevice.java

```diff
@@ -0,0 +1,94 @@
+package org.astarteplatform.devicesdk.tests.e2etest;
+
+import java.io.IOException;
+import java.util.*;
+import org.astarteplatform.devicesdk.AstarteDevice;
+import org.astarteplatform.devicesdk.protocol.*;
+import org.astarteplatform.devicesdk.tests.e2etest.utilities.AstarteHttpRequest;
+import org.astarteplatform.devicesdk.tests.e2etest.utilities.GenericDeviceMockData;
+import org.astarteplatform.devicesdk.tests.e2etest.utilities.GenericDeviceMockDataFactory;
+import org.astarteplatform.devicesdk.tests.e2etest.utilities.GenericMockDevice;
+import org.astarteplatform.devicesdk.transport.AstarteTransportException;
+import org.joda.time.DateTime;
+
+public class GenericAggregateDevice implements AstarteAggregateDatastreamEventListener {
+  private final GenericDeviceMockData m_deviceMockData;
+  private final GenericMockDevice m_mockDevice;
+  private final AstarteDevice m_astarteGenericDevice;
+
+  private final AstarteHttpRequest astarteHttpRequest;
+
+  public GenericAggregateDevice() throws Exception {
+    m_deviceMockData = new GenericDeviceMockDataFactory();
+    m_astarteGenericDevice = m_deviceMockData.getAstarteDeviceSingleton();
+    m_mockDevice = m_deviceMockData.getMockData();
+    astarteHttpRequest = new AstarteHttpRequest();
+  }
+
+  public void aggregateFromDeviceToServer()
+      throws AstarteInvalidValueException, AstarteInterfaceMappingNotFoundException,
+          AstarteTransportException, IOException, InterruptedException {
+
+    System.out.println(
+        "Test for aggregated object datastreams" + " in the direction from device to server.");
+    String interfaceName = m_mockDevice.getInterfaceDeviceAggr();
+
+    AstarteDeviceAggregateDatastreamInterface astarteDeviceAggregateDatastream =
+        (AstarteDeviceAggregateDatastreamInterface)
+            m_astarteGenericDevice.getInterface(interfaceName);
+
+    astarteDeviceAggregateDatastream.streamData(
+        "/seco", m_mockDevice.getMockDataDictionary(), DateTime.now());
```

Comment:
Please remove all company refs from the code, use a generic name, like `sensor_id` `data` etc
```suggestion
        "/sensor_id", m_mockDevice.getMockDataDictionary(), DateTime.now());
```